CA-Policy.md

```diff
@@ -33,10 +33,14 @@ The main assessment procedures used are the [Federal Risk and Authorization Mana
 
 Assessments of 18F operations are performed in tandem with vulnerability scanning, malicious user testing, insider threat assessments, and other tests regularly conducted by the following teams: the cloud.gov team, TTS Technology Portfolio, GSA Information Security, and a 3PAO. The system is also under continuous monitoring from cloud.gov's Cloud Operations team.
 
-18F takes any results seriously, and it implements remediations as soon as possible. Plans of action and milestones (POAMs) are maintained to ensure any findings are resolved, compensated for, or accepted as an operational requirement.
+18F takes any results seriously, and it remediates issues as soon as possible. Plans of action and milestones (POA&Ms) are maintained to ensure any findings are resolved, compensated for, or accepted as an operational requirement.
```

Comment:
Should this still say `18F`?

Reply:
I've gone through and updated that a bunch of places now.